client/src/components/NotesList.tsx

```diff
@@ -0,0 +1,19 @@
+import AddNote from "./AddNote";
+import Note from "./Note";
+import Search from "./Search";
+
+const notesList = () => {
+  return (
+    <div className="notes-list">
+      {/* <Search /> */}
+      <AddNote />
+      <Note />
+      <Note />
+      <Note />
+      <Note />
+      <Note />
```

Comment:
try to use `notes.map(....)` for rendering multiple notes.

Reply:
Okay